RoomsEyeView/Logic/vtkSlicerRoomsEyeViewModuleLogic.cxx

```diff
@@ -389,6 +389,28 @@ vtkSlicerRoomsEyeViewModuleLogic::~vtkSlicerRoomsEyeViewModuleLogic()
 void vtkSlicerRoomsEyeViewModuleLogic::PrintSelf(ostream& os, vtkIndent indent)
 {
   this->Superclass::PrintSelf(os, indent);
+
+  // Transforms
+  os << indent << "Transforms:" << std::endl;
+  vtkSmartPointer<vtkMatrix4x4> matrix = vtkSmartPointer<vtkMatrix4x4>::New();
+  for (auto& transformPair : IECLogic->vtkSlicerIECTransformLogic::GetIecTransforms())

```

Comment:
Please do not have variables like `IECLogic` just like this in the code. It is an error that can mean two different things and make the code less readable. See similar comment above.